# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/tests/crashtests/1758127-1.html
# Commit: 42a5cf2bfccd
# Full Hash: 42a5cf2bfccdcc633e2ab80dcb7aa9f58bc87998
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2022-03-11 21:51:10
# Description:
#   Bug 1758127 - Don't use tight bounding rect if glyph transform fails. r=jnicol
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D140746
# ==============================================================================

diff -r da54c46e2e6d -r 42a5cf2bfccd gfx/tests/crashtests/1758127-1.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/tests/crashtests/1758127-1.html	Fri Mar 11 08:06:39 2022 +0000
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <style>
+      * {
+          display: table-footer-group;
+          transform: rotateY(18.03772675170004deg);
+          perspective: 116in;
+          letter-spacing: -80ex;
+      }
+  </style>
+</head>
+</html>
\ No newline at end of file